shell/gpu/gpu_surface_metal_impeller.mm

```diff
@@ -99,6 +99,10 @@
 
   auto drawable = impeller::SurfaceMTL::GetMetalDrawableAndValidate(
```

Comment:
This can be nullptr, see https://github.com/flutter/engine/blob/47465eb4fbe6e01285172e7ec358521907828c5e/impeller/renderer/backend/metal/surface_mtl.mm#L26-L39